### PR TITLE
Merge MultiAlign

### PR DESCRIPTION
**Pull request by cnoahris**

### CODE REVIEW

**Comment by HokageM** [comment]
Cool! Great work!

**Comment by HokageM** [comment]
I would suggest to Renate the namespace to coaler::multialign

**Review by HokageM — APPROVED** [review]

**Comment by HokageM** [comment]
Please remove header with autor/date. Let us define a coaler header with MIT license

File: src/multialign/MultiAlignerResult.hpp [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#pragma once
+#include <unordered_map>
+#include "BasicClasses/Forward.hpp"
+
+namespace MultiAlign
+{
+    struct MultiAlignerResult{
+        double score;
+        std::unordered_map<LigandID, PoseID> poseIDsByLigandID;
+    };
```
Comment (HokageM) [inline src/multialign/MultiAlignerResult.hpp:14]: std::vector<MultiAlign::Ligand>& ligands

We need this vector here for the OutputWriter. To access molecules by ID.

File: src/multialign/BasicClasses/Ligand.hpp [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        unsigned getNofPoses() const noexcept;
+
+        /**
+         *
+         * @return The molecule represented by the ligand.
+         */
+        RDKit::RWMol getMolecule() const noexcept;
+
+    private:
+        LigandID m_id;
```
Comment (mariusrueve) [inline src/multialign/BasicClasses/Ligand.hpp:54]: Why are those variables called m_ ? I learned different naming convention

Comment (cnoahris) [reply src/multialign/BasicClasses/Ligand.hpp:54]: thats AMD research group style. If its okay for you id appreciate if we can stick with it. lots of m_ members already defined and kind of habit :)

Comment (nikplx) [reply src/multialign/BasicClasses/Ligand.hpp:54]: Please just make it uniform

**Pull request closed without merging**